shared/css/buttons.scss

```diff
@@ -5,7 +5,7 @@
   background-color: $color;
   border-color: $color;
   color: $white;
-  margin: 0;
+  margin: 0px;
```

Comment:
This file shouldn't be linted based on our `scss_files` setting: https://github.com/code-dot-org/code-dot-org/blob/staging/.scss-lint.yml#L1